examples/oneapi/cpp/source/decision_forest/df_cls_dense_batch.cpp

```diff
@@ -18,50 +18,51 @@
 #include "oneapi/dal/algo/decision_forest.hpp"
 #include "oneapi/dal/io/csv.hpp"
 
-using namespace oneapi;
+namespace onedal = oneapi::dal;
 namespace df = oneapi::dal::decision_forest;
 
 int main(int argc, char const *argv[]) {
-  const std::string train_data_file_name  = get_data_path("df_classification_train_data.csv");
-  const std::string train_label_file_name = get_data_path("df_classification_train_label.csv");
-  const std::string test_data_file_name   = get_data_path("df_classification_test_data.csv");
-  const std::string test_label_file_name  = get_data_path("df_classification_test_label.csv");
+    const std::string train_data_file_name = get_data_path("df_classification_train_data.csv");
+    const std::string train_label_file_name = get_data_path("df_classification_train_label.csv");
+    const std::string test_data_file_name = get_data_path("df_classification_test_data.csv");
+    const std::string test_label_file_name = get_data_path("df_classification_test_label.csv");
 
-  const auto x_train = dal::read<dal::table>(dal::csv::data_source{train_data_file_name});
-  const auto y_train = dal::read<dal::table>(dal::csv::data_source{train_label_file_name});
+    const auto x_train =
+        onedal::read<onedal::table>(onedal::csv::data_source{ train_data_file_name });
+    const auto y_train =
+        onedal::read<onedal::table>(onedal::csv::data_source{ train_label_file_name });
 
-  const auto x_test = dal::read<dal::table>(dal::csv::data_source{test_data_file_name});
-  const auto y_test = dal::read<dal::table>(dal::csv::data_source{test_label_file_name});
+    const auto x_test =
+        onedal::read<onedal::table>(onedal::csv::data_source{ test_data_file_name });
+    const auto y_test =
+        onedal::read<onedal::table>(onedal::csv::data_source{ test_label_file_name });
 
-  const auto df_desc =
-      df::descriptor<float, df::task::classification, df::method::dense>{}
-          .set_class_count(5)
-          .set_tree_count(10)
-          .set_features_per_node(1)
-          .set_min_observations_in_leaf_node(8)
-          .set_min_observations_in_split_node(16)
-          .set_min_weight_fraction_in_leaf_node(0.0)
-          .set_min_impurity_decrease_in_split_node(0.0)
-          .set_variable_importance_mode(df::variable_importance_mode::mdi)
-          .set_error_metric_mode(df::error_metric_mode::out_of_bag_error)
-          .set_infer_mode(df::infer_mode::class_labels | df::infer_mode::class_probabilities)
-          .set_voting_mode(df::voting_mode::weighted);
+    const auto df_desc =
+        df::descriptor<float, df::task::classification, df::method::dense>{}
```

Comment:
What are the default parameter values for the second and third template parameters of df? We can make it simpler if here the default values are used